hack/verify-generated-protobuf.sh

```diff
@@ -35,7 +35,7 @@ trap "cleanup" EXIT SIGINT
 cleanup
 for APIROOT in ${APIROOTS}; do
   mkdir -p "${_tmp}/${APIROOT}"
-  cp -a -T "${KUBE_ROOT}/${APIROOT}" "${_tmp}/${APIROOT}"
+  cp -a "${KUBE_ROOT}/${APIROOT}"/* "${_tmp}/${APIROOT}"
```

Comment:
Is it more explicit to have a trailing slash on the target directory?  This stuff is way more subtle than it should be.